README.md

```diff
@@ -1,27 +1,63 @@
 <div align="left">
         <img height="0" width="0px">
-        <img width="20%" src="/poweredbypysat.png" alt="pysat" title="pysat"</img>
+        <img width="20%" src="docs/images/pysatModels.png" alt="pysatModels" title="pysatModels"</img>
 </div>
 
-# pysatModelUtils
-[![Build Status](https://travis-ci.org/pysat/pysatModelUtils.svg?branch=main)](https://travis-ci.org/pysat/pysatModelUtils)
-[![Coverage Status](https://coveralls.io/repos/github/pysat/pysatModelUtils/badge.svg?branch=master)](https://coveralls.io/github/pysat/pysatModelUtils?branch=main)
+# pysatModels
+[![Documentation Status](https://readthedocs.org/projects/pysatmodels/badge/?version=latest)](https://pysatmodels.readthedocs.io/en/latest/?badge=latest)
```

Comment:
This currently shows as Failing. I don't know why. Everything looks ok. Compared to pysatMissions which shows green. Same style.... Could very well be that this pull just needs to be merged for the system and badges to catch up. Don't have anything specific to change. Noting I looked at it and current status.

Reply:
This is linked to latest, which is not the version created by this PR.